src/api/index.js

```diff
@@ -80,6 +81,17 @@ module.exports = function(config_hash) {
       });
     });
   }
+  // register middleware plugins
+  const plugin_params = {
+    config: config,
+    logger: Logger.logger,
+  };
+  const plugins = load_plugins(config, config.middlewares, plugin_params, function(p) {
```

Comment:
I'd guess `p` means plugin. I think a better naming here is possible.

Reply:
@juanpicado Both these have been fixed and a new commit have been pushed. Let me know if you want me to squash the commits.